impactlab_tools/os/config.py

```diff
@@ -0,0 +1,24 @@
+import sys
+import yaml
+
+def getConfigDictFromFile(filePath):
+    with open(filePath, 'r') as f:
+        confDict = yaml.load(f)
+    return confDict
+
+def getConfigDictFromSysArgv():
+    with open(sys.argv[1], 'r') as f:
+        confDict = yaml.load(f)
+
+        for key in serverConfig:
```

Comment:
Is serverConfig from global? Can you just call getConfigDictFromFile with a default argument instead of relying on a global value?